### PR TITLE
Adding settings to RPCWallet for MaxGasInvoke limit from Settings

### PR DESCRIPTION
**Pull request by vncoelho**
Closes https://github.com/neo-project/neo-cli/issues/318

### CODE REVIEW

File: RpcWallet/RpcWallet.cs [hunk 1]
```diff
@@ -127,7 +128,7 @@ private void ProcessInvoke(JObject result)
                     Script = result["script"].AsString().HexToBytes(),
                     Gas = Fixed8.Parse(result["gas_consumed"].AsString())
                 };
-                tx.Gas -= Fixed8.FromDecimal(10);
+                tx.Gas -= Settings.Default.MaxGasInvoke;
```
Comment (erikzhang) [inline RpcWallet/RpcWallet.cs]: Should be 10 gas (the free 10 gas for each transaction)

Comment (vncoelho) [reply RpcWallet/RpcWallet.cs]: In fact, I did not understand this part very well.

Comment (vncoelho) [reply RpcWallet/RpcWallet.cs]: What is the best way to limit the `Wallet.MakeTransaction(tx);` gas limit simulation?

Comment (erikzhang) [reply RpcWallet/RpcWallet.cs]: It created an InvocationTransaction for the `sendrawtransaction`.

Comment (vncoelho) [reply RpcWallet/RpcWallet.cs]: Maybe add an extra parameter on the https://github.com/neo-project/neo/blob/cc0f315854a9cf09f2efda2f83cf1199c9ece4a9/neo/Wallets/Wallet.cs#L223?

Comment (vncoelho) [reply RpcWallet/RpcWallet.cs]: Or just try to add simulate that to  tx.SystemFee = Settings.Default.MaxGasInvoke;

Comment (vncoelho) [reply RpcWallet/RpcWallet.cs]: I still did not understand the direction to implement this.

Comment (vncoelho) [reply RpcWallet/RpcWallet.cs]: I think I will leave this closed, maybe this can be solved direction on Neo RpcServer.cs class.

**Pull request closed without merging**